attribute_set/migrations/14.0.1.3.1/pre-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Akretion
+# @author: Raphaël Reverdy <raphael.reverdy@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    # 14.0.1.3.0 adds field company_dependent
+    # but ['ir.model']._instanciate_attrs is
+    # run before the creation of the column
+    # by the ORM
+    cr.execute(
+        "ALTER TABLE attribute_attribute ADD COLUMN IF NOT EXISTS company_dependent boolean"
```

Comment:
```suggestion
        "ALTER TABLE attribute_attribute ADD COLUMN IF NOT EXISTS company_dependent bool"
```
in postgres is bool

Reply:
`bool` is an alias for `boolean`.

>8.6. Boolean Type
>
> PostgreSQL provides the standard SQL type boolean; 

https://www.postgresql.org/docs/14/datatype-boolean.html
https://www.postgresql.org/docs/14/datatype.html